sdks/java/maven-archetypes/examples/src/main/resources/archetype-resources/pom.xml

```diff
@@ -151,7 +151,7 @@
     <dependency>
       <groupId>com.google.apis</groupId>
       <artifactId>google-api-services-pubsub</artifactId>
-      <version>v1-rev7-1.21.0</version>
+      <version>v1-rev7-1.22.0</version>
```

Comment:
Note: this jar does not exist and I am hoping that, per @davorbonaci 's claims, there will be a test failure. It did not happen locally with `mvn clean verify`.